FSharpLu.Json/Scripts/load-project-debug.fsx

```diff
@@ -0,0 +1,9 @@
+// Warning: generated file; your changes could be lost when a new file is generated.
```

Comment:
Uhh, I'm a little confused - I don't know what triggers generating those scripts :) Should I delete all of them and add .gitignore entry, or just add entry?